modin/pandas/test/test_rolling.py

```diff
@@ -78,6 +77,11 @@ def test_dataframe(data, window, min_periods, win_type):
             pandas_rolled.aggregate([np.sum, np.mean]),
         )
         df_equals(modin_rolled.quantile(0.1), pandas_rolled.quantile(0.1))
+        # `Rolling.counts` has a buggy side-effect on other rolling functions described in:
+        # https://github.com/pandas-dev/pandas/issues/39554
+        # So the testing of `.count` should always be the last until this bug
+        # will be fixed in pandas, to avoid this side-effect
+        df_equals(modin_rolled.count(), pandas_rolled.count())
```

Comment:
This bug is already fixed (pandas-dev/pandas#39554), so we need to revert this workaround when updating pandas to 1.2.2

Reply:
Definitely.